docs/resources/dns_recordset_v2.md

```diff
@@ -54,6 +54,11 @@ The following arguments are supported:
 * `value_specs` - (Optional) Map of additional options. Changing this creates a
   new record set.
 
+->
```

Comment:
add **Note:** pls

Reply:
There is no need, it's rendered with `Note` anyway:

![image](https://user-images.githubusercontent.com/8591561/105203417-2325ea80-5b54-11eb-9b3b-4aab64a57fea.png)